README.md

```diff
@@ -4,7 +4,7 @@ The easiest way to install Intercom in a rails app.
 
 For interacting with the Intercom REST API, use the `intercom` gem (https://github.com/intercom/intercom-ruby)
 
-Requires ruby 2.0 or higher for `intercom-rails  >= 0.4.0`
+Requires ruby 2.0 or higher for `intercom-rails  >= 0.4.1`
```

Comment:
I don't think this version number should be bumped; it's still true that 0.4.0 requires Ruby 2.0.

Maybe we shouldn't mention versions here at all?

```suggestion
Requires Ruby 2.0 or higher.
```